docs/configuration.md

```diff
@@ -0,0 +1,128 @@
+# Connector Configuration Reference
+
+## Configuring Named Connectors
```

Comment:
Let's add a section on "unnamed connectors" and how they work.

Reply:
Good idea. In progress.